chcemvediet/apps/accounts/backends.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import login
+from django.contrib.auth.backends import ModelBackend
+from django.core.urlresolvers import resolve
+
+from poleno.utils.http import get_request
+
+
+class AdminLoginAsBackend(ModelBackend):
+
+    def is_admin_path(self, path):
+        return resolve(path).func.__module__ in [
+                u'django.contrib.admin.options',
+                u'django.contrib.admin.sites',
+                u'adminplus.sites',
+                ]
+
+    def get_user(self, user_id):
```

Comment:
Strasne chaoticky mas to tu implementovane. Pls prepis bod po bode do pythonu, ako je to napisane v zadani https://github.com/martinmacko47/chcemvediet/issues/354#issuecomment-804325914

Reply:
:ok: